UnityGsdk/Assets/Tests/GSDKTests.cs

```diff
@@ -0,0 +1,413 @@
+using System;
+using System.IO;
+using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+using PlayFab;
+using PlayFab.MultiplayerAgent.Model;
+using PlayFab.MultiplayerAgent.Helpers;
+using static UnityEditor.Experimental.GraphView.GraphView;
+using System.Threading;
+using System.Threading.Tasks;
+
+public class GSDKTests
+{
+    private readonly string ConfigFilePath = Application.dataPath + "/Tests/testConfig.json";
+
+    private static SimpleJsonInstance _jsonInstance = new SimpleJsonInstance();
+
+    private GSDKConfiguration _testConfig;
+
+    [SetUp]
```

Comment:
how are these tests running? are we planning to add a GitHub action that runs them on PR?

Reply:
Yes, there's a PR pipeline in progress that will hook in here once it's done